src/dapla_pseudo/v1/models.py

```diff
@@ -136,16 +86,105 @@ def keyset_list(self) -> t.Union[t.List[PseudoKeyset], None]:
         return None if self.keyset is None else [self.keyset]
 
 
+class PseudoFunctionKeywordArgs(BaseModel):
+    """Representation of the possible keyword arguments"""
+
+    key_id: PredefinedKeys
+    strategy: t.Optional[UnknownCharacterStrategy] = None  # Only used for FF31
+    version_timestamp: t.Optional[str] = None  # Only used for 'map-sid'
```

Comment:
I think we should type this, for example with something from the datetime package, otherwise the user needs to guess the expected format.

Reply:
Maybe is should be called something else than `version_timestamp` - `version_identifier`, `version_key`, etc..